Jenkinsfile-JDK11

```
@@ -0,0 +1,210 @@
+#!groovy
+// Jenkinsfile for building a PR and running a subset of tests against it
+def pom
+def DOMAIN_NAME
+def payaraBuildNumber
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage('Report') {
+            tools {
+                jdk "zulu-8"
+            }
+            steps {
+                script{
+                    pom = readMavenPom file: 'pom.xml'
+                    payaraBuildNumber = "PR${env.ghprbPullId}#${currentBuild.number}"
+                    DOMAIN_NAME = "test-domain"
+                    echo "Payara pom version is ${pom.version}"
+                    echo "Build number is ${payaraBuildNumber}"
+                    echo "Domain name is ${DOMAIN_NAME}"
+                }
+            }
+        }
+        stage('Build') {
+            tools {
+                jdk "zulu-8"
+            }
+            steps {
+                echo '*#*#*#*#*#*#*#*#*#*#*#*#  Building SRC  *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
+                sh """mvn -B -V -ff -e clean install -PQuickBuild \
+                -Djavax.net.ssl.trustStore=${env.JAVA_HOME}/jre/lib/security/cacerts \
+                -Djavax.xml.accessExternalSchema=all -Dbuild.number=${payaraBuildNumber} \
+                -Dsurefire.rerunFailingTestsCount=2"""
+                echo '*#*#*#*#*#*#*#*#*#*#*#*#    Built SRC   *#*#*#*#*#*#*#*#*#*#*#*#*#*#*#'
+            }
+            post{
+                success{
+                    archiveArtifacts artifacts: 'appserver/distributions/payara/target/payara.zip', fingerprint: true
+                    archiveArtifacts artifacts: 'appserver/extras/payara-micro/payara-micro-distribution/target/payara-micro.jar', fingerprint: true
+                }
+            }
+        }
+        stage('Setup for Quicklook Tests') {
+            tools {
+                jdk "zulu-11"
+            }
+            steps {
+                sh "rm *.zip"
```

Build fails here

---

This seems to be trying to delete the log files from previous runs: https://github.com/payara/Payara/pull/4315/files#diff-28788dd8800bef08d2db40a4380a2f1eR71

The following change might work, haven't tried it within a Jenkinsfile (and not sure on the placement of the quotes, pick which you think is right):
```suggestion
                sh "rm *.zip || true"
```
```suggestion
                sh "rm *.zip" || true
```